CMakeLists.txt

```diff
@@ -122,7 +122,7 @@ install(TARGETS linalg EXPORT linalgTargets
 install(EXPORT linalgTargets
     FILE linalgTargets.cmake
     NAMESPACE std::
-    DESTINATION cmake
+    DESTINATION lib/cmake/stdBLAS
```

Comment:
Should be `lib/cmake/<ProjectName>` so `stdBLAS -> linalg`?

Reply:
Ah, yes, that's right, thanks for the catch!